test/cli/delegated_routing_http_test.go

```diff
@@ -17,6 +17,7 @@ func TestHTTPDelegatedRouting(t *testing.T) {
 
 	fakeServer := func(resp string) *httptest.Server {
 		return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
```

Comment:
This means we are not testing streaming :upside_down_face:  
Filled https://github.com/ipfs/kubo/issues/9873

Reply:
https://github.com/ipfs/kubo/pull/9874